pyproject.toml

```diff
@@ -20,7 +20,7 @@ authors = [
 ]
 requires-python = ">=3.9,<3.12"
```

Comment:
I suggest we remove upper-bound pinning in qadence, but leave it in PyQ.


If you want to keep it here, then bump to <3.13

Reply:
bumped